ui/src/components/community-form.tsx

```diff
@@ -270,6 +292,26 @@ export class CommunityForm extends Component<
     i.props.onCancel();
   }
 
+  showNameExplain() {
+    let element = document.getElementById("name_explain_div");
+    if (element.innerHTML == "") {
+      element.innerHTML = '<p class="ml-4 mb-2">' + i18n.t('name_explain') + '</p>';
+    }
+    else {
+      element.innerHTML = "";
+    }
+  }
+
+  showDisplayNameExplain() {
+    let element = document.getElementById("displayname_explain_div");
+    if (element.innerHTML == "") {
+      element.innerHTML = '<p class="ml-4 mb-2">' + i18n.t('display_name_explain') + '</p>';
+    }
+    else {
+      element.innerHTML = "";
+    }
+  }
```

Comment:
Bad bad bad, never inject html like this in react. Make the template in the tsx, and hide it using a boolean:

```
showDisplayExplanation = !showDisplayExplanation;

...

{this.showDisplayExplanation && (
<div ...
)}
```

But also, don't do that above. Just have the `name_explanation` in the tippy, it doesn't need a whole div explanation.